recipe/meta.yaml

```diff
@@ -24,9 +21,12 @@ build:
     - export PETSC_DIR=$PREFIX
     - {{ PYTHON }} -m pip -v install --no-deps .
   skip: true  # [win]
-  string: {{ scalar }}_h{{ PKG_HASH }}_{{ build }}
+  skip: true  # [osx and device != "host"]
+  string: {{ scalar }}_h{{ PKG_HASH }}_{{ build }}               # [device == 'host']
```

Comment:
let's remove the build string on the 4py packages, I don't think it provides anything

Reply:
unless users request `conda install petsc4py=*=*cuda*`

Reply:
I don't think it makes a lot of sense to request that. They can still do `petsc4py petsc=3.21=cuda*` with the sam eresult. Having it on the 4py package itself, too, doesn't add anything.